### PR TITLE
feat: Update Nova Keystone Jobs to match common architecture

### PR DESCRIPTION
**Pull request by intlabs**
**What is the purpose of this pull request?**: To bring Nova Keystone Jobs inline with other services.

**What issue does this pull request address?**: N/A

**Notes for reviewers to consider**:
This PR should bring Nova Keystone Jobs inline with Cinder and Heat, and is part of the effort to standardize the common elements in charts prior to moving to OpenStack.

**Specific reviewers for pull request**:
@alanmeadows @v1k0d3n @larryrensing @wilkers-steve

### CODE REVIEW

File: nova/templates/configmap-bin.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@ metadata:
...
 {{ tuple "bin/_db-sync.sh.tpl" . | include "helm-toolkit.template" | indent 4 }}
+  ks-service.sh: |+
+{{- include "helm-toolkit.keystone_service" . | indent 4 }}
+  ks-endpoints.sh: |+
+{{- include "helm-toolkit.keystone_endpoints" . | indent 4 }}
+  ks-user.sh: |+
+{{- include "helm-toolkit.keystone_user" . | indent 4 }}
   init.sh: |
 {{ tuple "bin/_init.sh.tpl" . | include "helm-toolkit.template" | indent 4 }}
   post.sh: |
```
Comment (intlabs) [inline nova/templates/configmap-bin.yaml]: Should be removed...

**Comment by intlabs** [comment]
@wilkers-steve thanks for the review :)

I need to do a little bit more cleanup for a couple of things I missed, I'll ping you soon as they are ready.

**Comment by intlabs** [comment]
@wilkers-steve I've made a few minor tweaks, could you please review this again at your convenience.

**Comment by larryrensing** [comment]
LGTM @intlabs

File: nova/values.yaml [hunk 2]
```diff
@@ -132,60 +134,48 @@ memcached:
 dependencies:
```
Comment (v1k0d3n) [inline nova/values.yaml]: thanks for your incredible patience while i've been fighting the good fight (day job/i.e. calls). you may have already addressed this in another PR that I'm about to get to soon, but QQ:

do we want dependency order to be the same across all charts (i.e. `db_init` > `db_sync` > `api` etc? i know you're in the middle of an awesome effort to clean this up, so you may have actually tacked this as part of another PR I'm about to review.

Comment (intlabs) [inline nova/values.yaml]: @v1k0d3n, we do :) I'm picking up the drips and drabs that have got through as part of the final job cleanup that's started with: https://github.com/att-comdev/openstack-helm/pull/289.

Comment (v1k0d3n) [inline nova/values.yaml]: sounds good. tracked on the roadmap now. thanks.

**Pull request merged**